### PR TITLE
Fix build on LLVM 10.0.0

### PR DESCRIPTION
**Pull request by OMantere**
Fixed some breaking changes to the 10.0.0 api.

Probably cannot be merged without breaking earlier versions but you might want to use this for LLVM 10 compatibility.

### CODE REVIEW

**Comment by mbrowne** [comment]
As someone who wanted to install this module, thank you for this PR. If anyone else would like to use it while waiting for this PR to be merged, I tagged it on github and you can install it like this:

```
npm i github:mbrowne/llvm-node#llvm-node-v2.1.1-gitpkg
```

File: src/ir/alloca-inst.cc [hunk 1]
```diff
@@ -68,7 +68,7 @@ NAN_SETTER(AllocaInstWrapper::setAlignment) {
     }
 
     auto* wrapper = AllocaInstWrapper::FromValue(info.Holder());
-    wrapper->getAllocaInst()->setAlignment(Nan::To<uint32_t>(value).FromJust());
+    wrapper->getAllocaInst()->setAlignment(llvm::MaybeAlign(Nan::To<uint32_t>(value).FromJust()));
```
Comment (MichaReiser) [inline src/ir/alloca-inst.cc:71]: Please branch the logic between LLVM < 10 and LLVM 10+

```
#if LLVM_VERSION_MAJOR < 10
        auto alignment = Nan::To<uint32_t>(value).FromJust();
    #else
        auto alignment = llvm::MaybeAlign(Nan::To<uint32_t>(value).FromJust());
    #endif

    wrapper->getAllocaInst()->setAlignment(alignment);
```

**Comment by MichaReiser** [comment]
Thanks for the PR and sorry for the late code review. I'm unfortunately not able to make changes to your PR. 

This looks good to me if we gate the change to LLVM 10+ only to maintain backward compatibility.

**Comment by OMantere** [comment]
@MichaReiser Added LLVM version branch.

**Comment by MichaReiser** [comment]
Thx

**Pull request merged**